conference/react-js/src/components/Header/index.jsx

```diff
@@ -1,31 +1,64 @@
 import React, { PureComponent } from 'react';
+import classNames from 'classnames';
 import SignInForm from '../../containers/SignInForm';
 
 class Header extends PureComponent {
   constructor() {
     super();
-
-    this.onButtonAccountClick = this.onButtonAccountClick.bind(this);
-    this.state = { visible: true };
+    this.state = {
+      visible: false,
+    };
   }
 
-  onButtonAccountClick() {
-    const currentState = this.state.visible;
-    this.setState({ visible: !currentState });
-  }
+  onButtonAccountClick = () => {
+    this.setState(
+      { visible: !this.state.visible },
```

Comment:
You don't understand my comment about callback for setState
https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous